src/actions.ts

```diff
@@ -0,0 +1,364 @@
+			},
+		},
+		clearAllAutoButtons: {
+			name: 'Clear All Auto Buttons',
+			description: 'Clear all auto button data. This action is only for debugging purposes.',
+			options: [],
+			async callback() {
+				clearAllAutoButtons(self)
+				self.checkFeedbacks('learnMode', 'autoPreset')
+			},
+		},
+		triggerMovement: {
+			name: 'Trigger move',
+			description:
+				'Execute a preset or random move on an auto-move director. To select a device, you first need to create a device in MiruSuite and add a video input to it. This action needs an auto-move director to be installed on the device.',
+			options: [
+				getDeviceSelector(self, videoDeviceOptions),
+				{
+					id: 'type',
+					type: 'dropdown',
+					label: 'Movement type',
+					choices: [
+						{ id: 'preset', label: 'Nearby preset' },
+						{ id: 'random', label: 'Random direction' },
+					],
+					default: 'preset',
+				},
+			],
+			async callback(event) {
+				const deviceId = Number(event.options.deviceId)
+				if (event.options.type === 'preset') {
+					self.log('info', 'Triggering preset transition move for device ' + deviceId)
+					await backend?.triggerPresetTransitionMove(deviceId)
+				} else if (event.options.type === 'random') {
+					self.log('info', 'Triggering random move for device ' + deviceId)
+					await backend?.triggerRandomMove(deviceId)
+				}
+			},
+		},
+		triggerReturnToHome: {
+			name: 'Return to home',
+			description:
+				'Return device to home position. To select a device, you first need to create a device in MiruSuite and add a video input to it. This action needs a PTZ controller to be installed on the device.',
+			options: [getDeviceSelector(self, videoDeviceOptions)],
+			async callback(event) {
+				const deviceId = Number(event.options.deviceId)
+				await backend?.triggerReturnToHome(deviceId)
+			},
+		},
+		exitSteadyMode: {
+			name: 'Exit steady mode',
+			description:
+				'Exit steady mode of device. To select a device, you first need to create a device in MiruSuite and add a video input to it. This action needs a head tracking director to be installed on the device.',
+			options: [getDeviceSelector(self, videoDeviceOptions)],
+			async callback(event) {
+				const deviceId = Number(event.options.deviceId)
+				await backend?.exitSteadyMode(deviceId)
+			},
+		},
+		stopAutoMove: {
+			name: 'Stop move',
+			description:
+				'Stop auto-movement of device. To select a device, you first need to create a device in MiruSuite and add a video input to it. This action needs an auto-move director to be installed on the device.',
+			options: [getDeviceSelector(self, videoDeviceOptions)],
+			async callback(event) {
+				const deviceId = Number(event.options.deviceId)
+				await backend?.stopAutoMove(deviceId)
+			},
+		},
+		toggleAutoCut: {
+			name: 'Toggle Auto Cut',
+			description: 'Enable or disable Auto cut. You first need to correctly setup AutoCut in MiruSuite.',
+			options: [],
```

Comment:
Instead of only a toggle action it would be better to have a autocut action with the options of on/off/toggle